src/style.css

```diff
@@ -1 +1,69 @@
-/* add styles here */
+body {
+  margin: 0 8px;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
According to the checklist, you should use fallback fonts - alternative font-family in case the main one doesn't work. The fallback font should be a similar style to the main font, but more common. It's usually a good idea to have at least one fallback font. In your case, you could add 'Helvetica' as a fallback font after 'Avenir'.